WORKSPACE

```diff
@@ -26,7 +26,7 @@ load("@com_google_api_gax_java//:repositories.bzl", "com_google_api_gax_java_rep
 
 com_google_api_gax_java_repositories()
 
-_googleapis_commit = "44d6bef0ca6db8bba3fb324c8186e694bcc4829c"
+_googleapis_commit = "7438480b2a1bc6371d748e974f7a3647f90c4e8d"
```

Comment:
Updating the googleapis commit to a later commit (It'll be from Jan 12th, 2023). I'm not sure if there is significance of this particular commit, but I'm using apigee for some golden integration tests and the grpc+rest transports for the library was enabled after this commit

Reply:
If this commit can be updated, perhaps it would make sense to do this in a separate PR? This would require updating the goldens for every existing library.

Reply:
This been updated and merged in https://github.com/googleapis/gapic-generator-java/pull/1249